pkg/resolver/related_test.go

```diff
@@ -104,7 +106,7 @@ func Test_SearchResolver_RelatedKindsRelationships(t *testing.T) {
 	config.Cfg.RelationLevel = 3
 
 	var resultList []*string
-
+	propTypesMock := map[string]string{"uid": "string"}
```

Comment:
Does the test need uid to be in the map?

Reply:
I don't think it's necessary for these tests. The related resources does not directly depend on property types (only initial query does)

Reply:
Removing from tests.